src/components/pages/ReportForm/index.vue

```diff
@@ -0,0 +1,87 @@
+<template lang="pug">
+  .issue
+    .container(v-if="project")
+      h2 {{ $t('title', { project: project.name }) }}
+
+      form-wizard(@on-complete='onComplete', 
+                  color='#00A1DF',
+                  title="", subtitle="",
+                  :back-button-text="$t('buttons.back')",
+                  :next-button-text="$t('buttons.next')",
+                  :finish-button-text="$t('buttons.finish')")
+
```

Comment:
Try not to leave this spaces, since pug is an indentation language it makes the code not fully readable.